### ISSUE TITLE
Exception when Building WaPo v3 Index

### ISSUE THREAD

**Issue by PepijnBoers** [castorini/anserini#1291]
I'm building the index for the 3th version of the Washington Post collection, which adds 154,418 new documents from 2018 and 2019. However, I encountered an error while doing so:

```
ERROR [pool-2-thread-1] index.IndexCollection$LocalIndexerThread (IndexCollection.java:241) - pool-2-thread-1: Unexpected Exception:
java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing required creator property ‘published_date’ (index 3)
```

Indicating that the specific article **did not possess a publication date**. The only date available is in the `contents` section as: `{"content": 1514865008000, "mime": "text/plain", "type": "date"}`, the  usual `"published_date": 1325376562000` outside `contents` is missing.

This is the case for at least 2 articles, but I suspect there to be more.

I used the following command line option to build the Index:
```
./target/appassembler/bin/IndexCollection -collection WashingtonPostCollection \
 -input /Volumes/Samsung_T5/WashingtonPost.v3/data/ -generator WashingtonPostGenerator \
 -index lucene-index.core18.pos+docvectors+rawdocs_all+contents \
 -threads 1 -storePositions -storeDocvectors -storeRaw -storeContents -optimize 
```

**Comment by ChanikanWitt** [comment]
Hi, I am wondering whether this issue is solved yet. I try building index in WashingtonPost v3 and still had the same issue.

**Comment by PepijnBoers** [comment]
Hi @ChanikanWitt this issue has been resolved on June 21 and was included in the v0.9.4 release. Is the build you're using up to date?

**Comment by ChanikanWitt** [comment]
Oh okay. It works now. Thank you!
